### PR TITLE
Update NAS.netkan

### PR DESCRIPTION
**Pull request by zzhwudiaa**
Switched dependant to BDArmoryContinued and removed unnecessary dependants

### CODE REVIEW

**Comment by linuxgurugamer** [comment]
#rebuild

**Comment by politas** [comment]
Crashes my CKAN client when I try to install it.

**Comment by linuxgurugamer** [comment]
There is a bad download: Download Failure

A corrupted download was found.
Indication 1

**Comment by linuxgurugamer** [comment]
#rebuild

**Comment by linuxgurugamer** [comment]
The downloaded file from Spacedock is:

NAS_-_Naval_Artillery_System-0.6.0.6.zip

but it seem sto be looking fo r0.6.1.1

**Comment by politas** [comment]
#rebuild

**Comment by politas** [comment]
My local test gave a download link of https://spacedock.info/mod/83/NAS%20-%20Naval%20Artillery%20System/download/0.6.0.6

File: NetKAN/NAS.netkan [hunk 1]
```diff
@@ -8,10 +8,6 @@
     "license"      : "CC-BY-SA-4.0",
     "x_netkan_force_v": true,
```
Comment (politas) [inline NetKAN/NAS.netkan]: To fix up the ordering issue, add this:

    "x_netkan_epoch": 1,

Comment (Olympic1) [inline NetKAN/NAS.netkan]: pinging @zzhwudiaa

**Pull request closed without merging**